5.020.001-64bit/Dockerfile

```diff
@@ -11,6 +11,12 @@ RUN curl -SL https://cpan.metacpan.org/authors/id/S/SH/SHAY/perl-5.20.1.tar.bz2
     && sha1sum -c /tmp/sha1.txt \
     && tar --strip-components=1 -xjf perl-5.20.1.tar.bz2 -C /usr/src/perl \
     && rm perl-5.20.1.tar.bz2 /tmp/sha1.txt
+    && ./Configure -Duse64bitall  -des \
+    && make -j$(nproc) \
+    && TEST_JOBS=$(nproc) make test_harness \
+    && make install \
+    && cd /root
+    && rm -fr /usr/src/perl
```

Comment:
COPY has depressing Docker cache behavior, so I'd recommend sticking to the
multiline RUN even though it's not exactly pretty